database/responses.go

```diff
@@ -8,3 +8,18 @@ type ProjectThumbail struct {
 	ProgramYear int    `json:"programYear"`
 	ProgramTerm string `json:"programTerm"`
 }
+
+type ProjectDetails struct {
+	ProjectID      string   `json:"projectId"`
+	OrganizationID string   `json:"orgId"`
+	LFXProjectUrl   string   `json:"lfxProjectUrl"`
+	Name           string   `json:"name"`
+	Industry       string   `json:"industry"`
+	Description    string   `json:"description"`
```

Comment:
Run `go fmt` on this file to format it correctly.

Reply:
Done, prettier was not working with auto-save ON, now it is formatted correctly.